scripts/qa_crowbarsetup.sh

```diff
@@ -1305,6 +1307,30 @@ function onadmin_repocleanup()
     zypper mr -d sp3sdk
 }
 
+# manila-share service (in combination with the generic driver) needs to
+# access (via ssh) the service-instance (which is a nova instance) to
+# configure the shares (i.e. export a nfs share)
+function crowbar_create_network_manila()
```

Comment:
at some point we might want to add that to the default shipped network.json perhaps?